packages/exo/src/exo-makers.js

```diff
@@ -130,9 +149,7 @@ export const defineExoClass = (
     if (finish) {
       finish(context);
     }
-    return /** @type {M & import('@endo/eventual-send').RemotableBrand<{}, M>} */ (
-      self
-    );
+    return self;
```

Comment:
is this still typed? consider adding a type regression test

Reply:
Yes, still typed.  The types of these `define*` methods are exercised in `test-heap-classes.js`.